src/v2/services/implementations/PolkadotWalletService.ts

```diff
@@ -77,6 +77,14 @@ export class PolkadotWalletService extends WalletService implements IWalletServi
           tip = tip ? ethers.utils.parseEther(tip).toString() : '1';
         }
 
+        const useableBalance = await this.assetsRepository.getNativeBalance(senderAddress);
+        const message = 'Minimum balance on the network is 0.01';
```

Comment:
Let's add it [here](https://github.com/AstarNetwork/astar-apps/blob/4718312b815cbb895dc08c19bbd249b397ffd535/src/modules/toast/index.ts#L1)